multitenancy-spring-boot-autoconfigure/src/main/java/org/axonframework/extensions/multitenancy/autoconfig/TenantPersistentStreamMessageSourceFactory.java

```diff
@@ -0,0 +1,55 @@
+package org.axonframework.extensions.multitenancy.autoconfig;
+
+import io.axoniq.axonserver.connector.event.PersistentStreamProperties;
+import org.axonframework.axonserver.connector.AxonServerConfiguration;
+import org.axonframework.axonserver.connector.event.axon.PersistentStreamMessageSource;
+import org.axonframework.config.Configuration;
+import org.axonframework.extensions.multitenancy.components.TenantDescriptor;
+
+import java.util.concurrent.ScheduledExecutorService;
+
+/**
+ * Factory interface for creating a PersistentStreamMessageSource for a given tenant.
+ * <p>
+ * This interface is used to create a PersistentStreamMessageSource for a given tenant, which is represented by a {@link TenantDescriptor}.
+ Sure, here is the JavaDoc for the `TenantPersistentStreamMessageSourceFactory` interface:
```

Comment:
Looks like something went wrong here... There is markdown in between two Javadoc sections.